tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/RecoveryResource.java

```diff
@@ -115,13 +115,13 @@ public Response phase2(@HeaderParam(LRA_HEADER) URI lraId) {
 
         // assert compensate has been called
         int compensations = lraMetricService.getMetric(LRAMetricType.Compensated, lraId, RecoveryResource.class.getName());
-        if (compensations != 1) {
+        if (compensations < 1) {
```

Comment:
compensations == 0 is more efficient (and readable)

Reply:
`LRAMetricService$getMetric` can return -1 if the Compensate metrics is not incremented.

Reply:
I missed that.

Reply:
Ah yes, -1 is a valid return code.